python-checks/src/main/java/org/sonar/python/checks/cdk/CdkUtils.java

```diff
@@ -87,30 +88,41 @@ public static Optional<DictionaryLiteral> getDictionary(Expression expression) {
     return Optional.empty();
   }
 
-  /**
-   * Resolve a particular argument of a call by keyword or get an empty optional if the argument is not set nor resolvable.
-   */
-  protected static Optional<ExpressionFlow> getArgument(SubscriptionContext ctx, CallExpression callExpression, String argumentName) {
-    return getArgument(ctx, callExpression, TreeUtils.argumentByKeyword(argumentName, callExpression.arguments()));
-  }
 
   protected static Optional<ExpressionFlow> getArgument(CallExpression callExpression, String argumentName) {
     // ctx not required as long as we don't try to raise an issue with ExpressionFlow
-    return getArgument(null, callExpression, TreeUtils.argumentByKeyword(argumentName, callExpression.arguments()));
+    return getArgument(null, callExpression, argumentName);
   }
 
-  private static Optional<ExpressionFlow> getArgument(SubscriptionContext ctx, CallExpression callExpression, RegularArgument regArg) {
+  /**
+   * Resolve a particular argument of a call by keyword or get an empty optional if the argument is not set nor resolvable.
+   */
+  protected static Optional<ExpressionFlow> getArgument(SubscriptionContext ctx, CallExpression callExpression, String argumentName) {
```

Comment:
As code analysis says that `ctx` should be marked with `@javax.annotation.Nullable` annotation